metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -1295,64 +1299,28 @@ def _extract_metadata_from_derived_table_sql(
         # Skip queries that contain liquid variables. We currently don't parse them correctly.
         # Docs: https://cloud.google.com/looker/docs/liquid-variable-reference.
         # TODO: also support ${EXTENDS} and ${TABLE}
-        if "{%" in sql_query:
```

Comment:
where does all the jinja-related logic go now?

Reply:
In this commit I updated the old code to as per syntax, will add complete code in next commit.